docs/source/development.rst

```diff
@@ -87,23 +87,54 @@ passed via the ``--kube-config`` and ``--kube-context`` arguments to pytest_.
 Furthermore, the context *must* start with either ``crate-`` or be called
 ``minikube``.
 
-.. note::
+Minikube
+^^^^^^^^
 
-   If you are using macOS in combination with Minikube, you need to set up a
-   `Minikube tunnel`_ in order to get an external IP address assigned to the
-   CrateDB services.
+Using minikube_ as Kubernetes backend for the tests requires a few
+preparations. Make sure your minikube has configured enough resources for
```

Comment:
A little more natural is "For example, make sure your Minikube...". Also think about whether you want to capitalize Minikube or not - either is fine but should be consistent

Reply:
I think lower case is correct. https://minikube.sigs.k8s.io/docs/

Reply:
If it's lower case, do you mean it should also be lower case in the title?